tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "@edx/typescript-config",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "dist",
+    "jsx": "react-jsx",
+    "lib": ["ES2021"],
```

Comment:
[inform] Hoping to contribute these upstream to `@edx/typescript-config`, as they are generally applicable to all MFEs, not just this one. See https://github.com/openedx/typescript-config/pull/12 for more details.